cypress/integration/jsTaskParseListDOM.spec.js

```diff
@@ -14,15 +14,14 @@ describe('Parse list app ', () => {
   });
 
   it('should be sorted by salary DSC', () => {
-
     cy.get('ul > li').then(($ul) => {
-      const list = [...$ul].map(
-          (row) => row.dataset.salary.replace('$', '').replace(',', ''));
+      const list = [...$ul].map((row) =>
+        row.dataset.salary.replace('$', '').replace(',', ''),
+      );
       let counter = 0;
 
       for (let i = 0; i < list.length; i++) {
         if (Number(list[i + 1]) <= Number(list[i])) {
```

Comment:
Be careful with accessing array elements out of bounds. 'list[i + 1]' may be undefined when 'i' is the last index.